src/gui/insEdit.cpp

```diff
@@ -2422,6 +2477,20 @@ void FurnaceGUI::drawInsEdit() {
 
             ImGui::EndTabItem();
           }
+          if (ImGui::BeginTabItem("Wavetable Macros")) {
```

Comment:
No!
This will complicate hardware playback and possibly yield very weird results......

Reply:
Especially the "Effect" macro. I think this would have to be discussed separately.